src/02_policy_log_analytics/01_allowed_sku.tf

```diff
@@ -0,0 +1,21 @@
+resource "azurerm_policy_definition" "log_analytics_allowed_sku" {
+  name                = "log_analytics_allowed_sku"
+  policy_type         = "Custom"
+  mode                = "Indexed"
+  display_name        = "PagoPA Log Analytics Workspace allowed sku"
```

Comment:
maybe name should be something like "PagoPA Log Analytics Workspace must be linked to dedicated cluster"